scripts/build-controller-release.sh

```diff
@@ -166,3 +178,16 @@ controller-gen rbac:roleName=$K8S_RBAC_ROLE_NAME paths=./... output:rbac:artifac
 mv $helm_output_dir/templates/role.yaml $helm_output_dir/templates/cluster-role-controller.yaml
 
 popd 1>/dev/null
+
+if [[ $ACK_GENERATE_OLM == "true" ]]; then
+    echo "Generating operator lifecycle manager bundle assets for $SERVICE"
+    # if the user has specified an olm config use that. Otherwise, use the default one
+    # expected in controller source.
+    DEFAULT_ACK_GENERATE_OLMCONFIG_PATH=$SERVICE_CONTROLLER_SOURCE_PATH/$SERVICE-olmconfig.yaml
+    ACK_GENERATE_OLMCONFIG_PATH=${ACK_GENERATE_OLMCONFIG_PATH:-$DEFAULT_ACK_GENERATE_OLMCONFIG_PATH}
+
+    olm_version=$(echo $RELEASE_VERSION | tr -d "v")
+    ag_olm_args="$SERVICE $olm_version -o $SERVICE_CONTROLLER_SOURCE_PATH --template-dirs $TEMPLATES_DIR --olm-config $ACK_GENERATE_OLMCONFIG_PATH --aws-sdk-go-version v1.35.5"
```

Comment:
Why are we hardcoding the aws sdk go version?

Reply:
The building of args for the `olm` subcommand is mirroring what's done on line 130/142 (building args for the `release` subcommand). From my perspective, I assumed it was needed given the current `TODO`.

Reply:
@komish is correct. You need to supply the --aws-sdk-go-version at the moment even for generation targets that don't use aws-sdk-go like OLM. This is due to the way the ack-generate root commands are set up. It is on my plate to get that fixed.